etc/nodes.conf

```diff
@@ -1,5 +1,16 @@
-WW_INTERNAL: 45
+WW_INTERNAL: 46
 nodeprofiles:
   default:
     comment: This profile is automatically included for each node
+    runtime overlay:
+      - generic
+    system overlay:
+      - wwinit
```

Comment:
These need to be updated to use the new split overlays, as "generic" no longer exists and there are more system overlays that should probably be included by default now besides "wwinit."